cmake/templates/pkgConfig.cmake.in

```diff
@@ -78,6 +78,15 @@ else()
   set(@PROJECT_NAME@_PREFIX ${@PROJECT_NAME@_INSTALL_PREFIX})
 endif()
 
+if(CMAKE_CROSSCOMPILING)
+  if("@DEVELSPACE@" STREQUAL "TRUE")
+    # can not disable host filesystem altogether; otherwise files in src/devel are no longer found
+    set(FIND_XXX_LOCATION_OPTION CMAKE_FIND_ROOT_PATH_BOTH)
```

Comment:
Ok. I was wondering though whether it is a good idea to allow CMake to search for header files & libraries in the host filesystem. As I mentioned as a comment it is necessary to find catkin files, but if this rule is applied to standard includes like `/usr/include` this might have undesired effects. I thought about explicitly searching the catkin workspace and not the host filesystem, but in case workspaces are chained this will get complicated fast.